src/containers/FriendsContainer.js

```diff
@@ -19,6 +19,7 @@ const FriendsContainer = props => {
     friendsCoordinates,
     user,
     history,
+    statuses,
   } = props
 
   const [directChannels, setDirectChannels] = useState([])
```

Comment:
Other option would be in the initial fetch and `fetchDirectChannelMembers` action get all the node user profiles for every friend, and then pass those as props further down to the `Friend` component to decide if some user has blocked this user (similar to direct channel members).  IMO this would make the friends page a bit more clear code-wise, but functionality is the same. And the downside is that the initial loading of friends page would get a little longer.

Reply:
I think I would prefer faster loading here, since probably blocking is not that common..? Or maybe it is :D